docs/source/configuration/overview.mdx

```diff
@@ -461,22 +508,22 @@ plugins:
 
 You can reference variables directly in your YAML file. This is useful for referencing secrets without including them in the file.
 
-Only environment variable expansion is currently supported, and variables must be prefixed with `env.`
+Currently, the Apollo Router supports expansion of environment variables and file paths. Corresponding variables must be prefixed with `env.` or `file.`
 
-Unix-style expansion is used. For example:
+The router uses Unix-style expansion. Here are some examples:
 
 - `${env.ENV_VAR_NAME}` expands to the value of environment variable `ENV_VAR_NAME`.
 - `${env.ENV_VAR_NAME:-some_default}` expands to the value of environment variable `ENV_VAR_NAME`, or falls back to the value `some_default` if the environment variable is not defined.
-- `${file.a.txt}` expands to the content of `a.txt`.
-- `${file.a.txt:-some_default}` expands to the content of `a.txt`, or falls back to the value `some_default` if the file does not exist.
+- `${file.a.txt}` expands to the contents of the file `a.txt`.
+- `${file.a.txt:-some_default}` expands to the contents of the file `a.txt`, or falls back to the value `some_default` if the file does not exist.
 
 Variable expansions are valid only for YAML _values_, not keys:
 
-```yaml {4,8} title="router.yaml"
+```yaml title="router.yaml"
```

Comment:
Until https://github.com/apollographql/router/issues/1374 has a solution, let's just remove this part and deal with the fact that it's not validated.

`````suggestion
<!-- TODO: Add back doc config validation when https://github.com/apollographql/router/issues/1374 is fixed -->
```yaml
````